src/lib_cxx/data/parser/cxx/utilityClang.cpp

```diff
@@ -85,6 +87,8 @@ SymbolKind utility::convertTagKind(const clang::TagTypeKind tagKind)
 		return SYMBOL_ENUM;
 	case clang::TTK_Interface:
 		return SYMBOL_KIND_MAX;
+	default:
+		return SYMBOL_UNKNOWN;
```

Comment:
I just checked the rest of the code base. It would be fine to return `SYMBOL_KIND_MAX` here as a default. That enum constant is already used for exactly this case.

Reply:
thank you, done